RealEstate/apps/core/views.py

```diff
@@ -121,10 +136,65 @@ def _homebuyer_post(self, request, homebuyer, *args, **kwargs):
         return render(request, self.homebuyer_template_name, context)
 
     def _realtor_get(self, request, realtor, *args, **kwargs):
-        return render(request, self.realtor_template_name, {})
+        couple = Couple.objects.filter(homebuyer__user=request.user)
```

Comment:
This is the realtor GET method so this query doesnt really make sense, can remove this line.